src/index.html

```diff
@@ -1,14 +1,89 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#home"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="main-nav">
+
+        <ul class="main-nav-list">
+          <li class="list-item">
+            <a
+              class="links is-active"
+              href="#"
+            >Apple</a>
```

Comment:
this is the proper styling
```suggestion
            <a class="links is-active" href="#">
              Apple
            </a>
```